### ISSUE TITLE
Unable to bring back old versions of templates

### ISSUE THREAD

**Issue by erickgonzalez** [dotCMS/core#28417]
### Parent Issue

#23366 

### Problem Statement

There isn't a way to restore an old version of a template.

### Steps to Reproduce

1. Create a new template
2. Edit the template and create a new version
3. Go to the history tab and click over a previous version
4. The Previous Version is shown, but there isn't a way to set it as the working one unless you make a change (this will trigger auto-save).

### Acceptance Criteria

There should be an option to set up the previous version as the Working one.

### dotCMS Version

Master (all LTSs)

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

**Comment by melissarojas-dotcms** [comment]
[Link to the design](https://www.figma.com/file/ECuBLIRWZjEzLzmlPrA0Rk/Template-Builder?type=design&node-id=1355%3A7653&mode=design&t=UpdSwIblYFqa9Iuj-1)

[Link to the prototype](https://www.figma.com/proto/ECuBLIRWZjEzLzmlPrA0Rk/Template-Builder?page-id=&type=design&node-id=1351-4214&viewport=-7050%2C-5017%2C0.31&t=NsKEt6apwIrUE5cD-1&scaling=scale-down&starting-point-node-id=1351%3A4214&mode=design)

**Comment by jdotcms** [comment]
I have confirmed the history does not have the bring back
in terms of the backend, FE can call this endpoint:
```
curl --location --request PUT 'http://localhost:8080/api/v1/versionables/9c68cef5-580c-40bb-8c38-41275ea33964/_bringback' 
```

The uuid is the INODE you want to get back

Thanks,
J

**Comment by zJaaal** [comment]
# IQA Passed 

Now we can Bring Back Templates

Docker Image: `[dotcms/dotcms:trunk_6616302]`


## Screenshot 

https://github.com/user-attachments/assets/1b4909f4-b164-4b63-9959-ef1fd3af72c5

**Comment by bryanboza** [comment]
Fixed, tested on the latest trunk image [761ef08] and now we are able to bring back without problems.
